apps/wallet/src/ui/app/components/nft-display/index.tsx

```diff
@@ -53,46 +54,65 @@ function NFTDisplayCard({
         </div>
     );
 
-    const defaultSection = (
-        <>
-            {expandable ? (
-                <div className={st.expandable}>
-                    <ExplorerLink
-                        type={ExplorerLinkType.object}
-                        objectID={nftObjectID}
-                        showIcon={false}
-                        className={st['explorer-link']}
-                    >
-                        View Image <Icon icon={SuiIcons.Preview} />
-                    </ExplorerLink>
-                </div>
-            ) : null}
-            {showlabel && displayTitle ? (
-                <div className={st.nftfields}>{displayTitle}</div>
-            ) : null}
-        </>
-    );
+    const defaultSection =
+        showlabel && displayTitle ? (
+            <div
+                className={cl(
+                    'items-center mt-2 text-steel-dark',
+                    animateHover &&
+                        'group-hover:text-black duration-200 ease-ease-in-out-cubic'
+                )}
+            >
+                {displayTitle}
+            </div>
+        ) : null;
 
+    const borderRadiusCl = borderRadius === 'md' ? 'rounded' : 'rounded-[10px]';
+    const borderRadiusHoverCl =
+        borderRadius === 'md' ? 'rounded-sm' : 'rounded-[5px]';
+    const mediaContainerCls = animateHover
+        ? `ease-ease-out-cubic duration-[400ms] group-hover:shadow-steel/50 group-hover:shadow-[0_0_20px_0] ${borderRadiusCl} hover:${borderRadiusHoverCl}`
+        : '';
+    const mediaCls = animateHover
```

Comment:
Similarly, I'd inline this. If you find it annoying to type the same classes in multiple places then it points to good componentization opportunity!